tests/api/resource_test.py

```diff
@@ -219,6 +219,7 @@ def setup_resource(self):
             scheduler=mock.Mock(),
             node_pool=mock.create_autospec(node.NodePool, ),
             max_runtime=mock.Mock(),
+            expected_runtime=mock.Mock(),
```

Comment:
Hmmm....., it should be...... I don't know actually. It looks like an attribute on the job_runs, but so does max_runtime? And it saves a couple of bytes for the return values of API by being an attribute on an individual job rather than on the job_runs.